src/core/transaction.js

```diff
@@ -1501,6 +1501,157 @@ Transaction.prototype.summary = function() {
   return txSummary;
 };
 
+/**
+ * Construct a 'SuspendedPaymentCreate' transaction
+ *
+ * Relevant setters:
+ *  - setSourceTag()
+ *  - setFlags()
+ *  - setDigest()
+ *  - setCancelAfter()
+ *  - setFinishAfter()
+ *
+ *  @param {String} source account
+ *  @param {String} destination account
+ *  @param {Amount} payment amount
+ */
+
+Transaction.prototype.suspendedPaymentCreate = function(options_) {
+  let options;
+
+  if (typeof options_ === 'object') {
```

Comment:
I don't think we need to support multiple calling syntaxes in new code. I think Brandon has been moving to object-only convention, so we should require that.